### ISSUE TITLE
Debugger attached. Waiting for the debugger to disconnect...

### ISSUE THREAD

**Issue by aman-compro** [Microsoft/vscode#42093]
- VSCode Version: Code 1.19.2 (490ef761b76b3f3b3832eff7a588aac891e5fe80, 2018-01-10T16:16:25.767Z)
- OS Version: Windows_NT ia32 10.0.10240
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
yaml|ada|0.0.9
ng-template|Ang|0.1.7
swagger-viewer|Arj|1.5.0
jquerysnippets|don|0.0.1
Angular2|joh|2.11.2
code-beautifier|mic|1.7.2
Angular-BeastCode|Mik|5.2.2
debugger-for-chrome|msj|4.0.0
Align|ste|0.2.0
vscode-js-console-utils|wht|0.7.0

(1 theme extensions excluded)

---

Steps to Reproduce:

1. Start Node Debugger
2. Error 
**Debugger attached.
Waiting for the debugger to disconnect...**

Earlier this issue was fixed when i restarted VS code , but now it is persisting even after i killed all the node processes

Reproduces without extensions: Yes/No

**Comment by isidorn** [comment]
@aman-compro what node version do you have?
Can you give us exact stpes on how to reproduce? Since I do not see this on my machine.
Can you also reproduce with latest vscode insiders https://code.visualstudio.com/insiders/

**Comment by aman-compro** [comment]
@isidorn I am using node v8.9.3.

There are no steps to replicate as such. Just pressing F5 to start debugging and getting this error (**Debugger attached. Waiting for the debugger to disconnect...**)

I reinstalled VS Code today and problem is still there.

and Yes I am getting this error with the latest VS code insider as well.

p.s: I have already checked there are no other node processes running on my system and also restarted the system just for making sure.
